nautobot/extras/templates/extras/jobqueue_retrieve.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load helpers %}
+
+{% block content_left_page %}
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <strong>Job Queue</strong>
+            </div>
+
+            <table class="table table-hover panel-body attr-table">
+                <tr>
+                    <td>Name</td>
+                    <td>
+                        <span>{{ object.name }}</span>
+                    </td>
+                </tr>
+                <tr>
+                    <td>Queue Type</td>
+                    <td>
+                        <span>{{ object.queue_type }}</span>
+                    </td>
+                </tr>
+                <tr>
+                    <td>Description</td>
+                    <td>
+                        <span>{{ object.description | placeholder }}</span>
+                    </td>
+                </tr>
+                <tr>
+                    <td>Tenant</td>
+                    <td>
+                        <span>{{ object.tenant | hyperlinked_object }}</span>
```

Comment:
Common pattern from other models:

```suggestion
                        {% if object.tenant and object.tenant.tenant_group %}
                            {{ object.tenant.tenant_group|hyperlinked_object }} /
                        {% endif %}
                        {{ object.tenant|hyperlinked_object }}
```